diff -u -r a/src/cpu/build68k.c b/src/cpu/build68k.c
--- a/src/cpu/build68k.c	2020-12-26 22:50:12.000000000 +0100
+++ b/src/cpu/build68k.c	2021-03-14 13:38:41.178048100 +0100
@@ -56,8 +56,16 @@
     }
 }
 
+#ifdef _WIN32
+INT WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance,
+    PSTR lpCmdLine, INT nCmdShow)
+{
+    int argc = __argc;
+    char **argv = __argv;
+#else
 int main(int argc, char **argv)
 {
+#endif
     int no_insns = 0;
 
     printf ("#include \"sysconfig.h\"\n");
diff -u -r a/src/cpu/gencpu.c b/src/cpu/gencpu.c
--- a/src/cpu/gencpu.c	2020-12-26 22:50:12.000000000 +0100
+++ b/src/cpu/gencpu.c	2021-03-14 13:38:50.480240700 +0100
@@ -9893,8 +9893,16 @@
 	postfix2 = -1;
 }
 
+#ifdef _WIN32
+INT WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance,
+	PSTR lpCmdLine, INT nCmdShow)
+{
+	int argc = __argc;
+	char **argv = __argv;
+#else
 int main(int argc, char *argv[])
 {
+#endif
 	init_table68k();
 
 	opcode_map =  xmalloc (int, nr_cpuop_funcs);
